tests/appsec/test_fingerprinting.py

```diff
@@ -17,14 +18,19 @@ def get_span_meta(r):
 @rfc("https://docs.google.com/document/d/1DivOa9XsCggmZVzMI57vyxH2_EBJ0-qqIkRHm_sEvSs/edit#heading=h.88xvn2cvs9dt")
 @features.fingerprinting
 class Test_Fingerprinting_Header_And_Network:
+    network_fingerprint_regex = r"net-[^-]*-[^-]*"
+    header_fingerprint_regex = r"hdr-[^-]*-[^-]*-[^-]*-[^-]*"
+
     def setup_fingerprinting_network(self):
         self.r = weblog.get("/", headers=ARACHNI_HEADERS)
         self.n = weblog.get("/")
 
     def test_fingerprinting_network(self):
         assert self.r.status_code == 200
         assert self.n.status_code == 200
-        assert all("_dd.appsec.fp.http.network" in m for m in get_span_meta(self.r))
+        r_span_meta = get_span_meta(self.r)
+        assert all("_dd.appsec.fp.http.network" in m for m in r_span_meta)
+        assert all(re.match(self.network_fingerprint_regex, m["_dd.appsec.fp.http.network"]) for m in r_span_meta)
```

Comment:
```suggestion
        assert all(re.match(self.network_fingerprint_regex, m["_dd.appsec.fp.http.network"]) for m in r_span_meta), f"{m["_dd.appsec.fp.http.network"]) = } does not match {network_fingerprint_regex} regex"
```

It would be good if we could get a better output than:
```
assert False
E        +  where False = all(<generator object Test_Fingerprinting_Header_And_Network.test_fingerprinting_network.<locals>.<genexpr> at 0x7f377cd136a0>)
```